core/app/models/spree/order.rb

```diff
@@ -400,7 +400,7 @@ def deliver_order_confirmation_email
 
     # Helper methods for checkout steps
     def paid?
-      payment_state == 'paid' || payment_state == 'credit_owed'
+      %w(paid credit_owed).include?(payment_state)
```

Comment:
@cbrunsdon that argument is a pretty slippery slope to me. At what point do you start to allow changes that you consider to result in "white noisy" merges?

While I can understand the mixed use convention, @mbj has offered a global change.

Putting off quality changes for the reason above seems tenuous at best.